filebeat/beater/filebeat.go

```diff
@@ -63,11 +67,33 @@ func New(b *beat.Beat, rawConfig *common.Config) (beat.Beater, error) {
 	return fb, nil
 }
 
+// Setup is called on user request (the -setup flag) to do the initial Beat setup.
+func (fb *Filebeat) Setup(b *beat.Beat) error {
```

Comment:
Yeah :). I didn't add it yet into the interface but I have a feeling that's where it will eventually get there.